docs/Video Streaming for Navigation Apps/Audio Streaming/index.md

```diff
@@ -27,8 +69,27 @@ The `SDLAudioStreamManager` will help you to do on-the-fly transcoding and strea
 sdlManager.streamManager?.audioManager.push(withFileURL: url)
 sdlManager.streamManager?.audioManager.playNextWhenReady()
 ```
+!@
+
+@![android]
+You can also send an Audio Source Uri to the `AudioStreamManager` to be played. To use it, replace the `pushResource` call in the example above to the `pushAudioSource` call shown below:
```

Comment:
As noted above, I don't think that the differences are defined well between these two APIs and I don't know that it's right to put resource in the giant block above and audio source down here. They're both files, right?